values/harbor/harbor.gotmpl

```diff
@@ -30,7 +30,7 @@ core:
     {{- end }}
 
   resources: {{- $h.resources.core | toYaml | nindent 4 }}
-  secret: {{ $h | get "core.secret" nil | quote }}
+  existingSecretKey: harbor-core-secret-key
```

Comment:
`core.existingSecretKey` does not exist in the Harbor helm chart. What was your intention while setting it?